Sources/ProjectAutomation/Target.swift

```diff
@@ -8,26 +8,36 @@ public struct Target: Codable, Equatable {
     /// The product type the target produces.
     public let product: String
 
+    /// The bundleId of the target.
+    public let bundleId: String
+
     /// List of file paths that are the target's sources.
     public let sources: [String]
 
     /// List of file paths that are the target's resources.
     public let resources: [String]
 
+    /// The target’s settings.
+    public let settings: ProjectAutomation.Settings
+
     /// The target’s dependencies.
-    public let dependencies: [TargetDependency]
+    public let dependencies: [ProjectAutomation.TargetDependency]
```

Comment:
We don't need the `ProjectAutomation.` as this can't depend on ProjectDescription or TuistGraph, right?

Reply:
But there is a mapping from TuistGraph.Project to ProjectAutomation.Project first

Reply:
I would keep things as they are in this PR, and evaluate the change in a separate one, as there are many more things to change in case